project/AppDependencies.scala

```diff
@@ -2,36 +2,36 @@ import sbt._
 object AppDependencies {
   import play.core.PlayVersion.current
 
-  private val silencerVersion = "1.7.9"
-  private val bootstrapPlayVersion = "6.4.0"
-  private val hmrcMongoPlayVersion = "0.68.0"
+  private val silencerVersion      = "1.7.9"
+  private val bootstrapPlayVersion = "7.2.0"
+  private val hmrcMongoPlayVersion = "0.71.0"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "3.22.0-play-28",
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc"         % "3.24.0-play-28",
     "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapPlayVersion,
     "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % hmrcMongoPlayVersion,
     "uk.gov.hmrc"                  %% "http-caching-client"        % "9.6.0-play-28",
-    "com.typesafe.play"            %% "play-json"                  % "2.9.2",
+    "com.typesafe.play"            %% "play-json"                  % "2.9.3",
     "uk.gov.hmrc"                  %% "play-json-union-formatter"  % "1.15.0-play-28",
     "org.typelevel"                %% "cats-core"                  % "2.8.0",
     "com.github.blemale"           %% "scaffeine"                  % "4.1.0",
```

Comment:
could be 5.2.1, if there is problems revert it to old version but worth a try

Reply:
breaks quite a lot ^^